terraform/modules/cluster/mig-with-container/cloudinit/main.tf

```diff
@@ -19,15 +19,8 @@ locals {
   _gcsfuse_host_mount   = "/tmp/cloud/gcsfuse_mnt"
 
   _startup_scripts_template_variables = {
-    script       = var.startup_script != null ? replace(var.startup_script, "\n", "\n    ") : ""
-    requirements = var.machine_has_gpu ? "aiinfra-install-gpu.service" : ""
-  }
-
-  cos_extensions_flags = var.cos_extensions_flags != null ? (
```

Comment:
cos_extensions_flags still exists in the cloudinit/variables.tf file.  Do you need to update that file too?

Reply:
that is on the "deleted" side!